packages/health/lib/src/health_factory.dart

```diff
@@ -402,6 +402,24 @@ class HealthFactory {
     return success ?? false;
   }
 
+  //Jasper implementation of getting specific data with limit quantity
+  Future<List<HealthDataPoint>> getDataWithLimit(List<HealthDataType> dataType, int limit) async {
+    List<HealthDataPoint> dataPoints = [];
+    final endTime = DateTime.now();
+    final startTime = DateTime(2000);
```

Comment:
Is there a reason for giving a time frame here? As far as I understand this is supposed to simply fetch data with a limit, so it should work exactly as the current implementation but with an extra parameter no?